XamarinCommunityToolkit/Views/StateLayout/StateLayout.shared.cs

```diff
@@ -0,0 +1,88 @@
+﻿using System;
+using System.Collections.Generic;
+using Xamarin.Forms;
+
+namespace Xamarin.CommunityToolkit.UI.Views
+{
+	public static class StateLayout
+	{
+		internal static readonly BindablePropertyKey StateViewsPropertyKey
+			= BindableProperty.CreateReadOnly("StateViews", typeof(IList<StateView>), typeof(Layout<View>), default(IList<StateView>),
+				defaultValueCreator: bindable =>
```

Comment:
Is there any sense to use defaultValueCreator here?
I guess we can just put ```new List<StateView>()``` as a default value
Or ```Enumerable.Empty<StateView>()``` in case we switch to IEnumerable interface